nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -94,6 +94,8 @@
 
 - `programs.gnupg.agent.pinentryFlavor` is now set in `/etc/gnupg/gpg-agent.conf`, and will no longer take precedence over a `pinentry-program` set in `~/.gnupg/gpg-agent.conf`.
 
+- OLIMEX Teres-A64 got Crust and U-Boot package declarations.
+
```

Comment:
This is not significant nor notable enough to get a release note.

The instructions in the PR body are not a requirement to always add, but a note to ensure you don't forget for when it is.

```suggestion
```